### PR TITLE
Making Ink Recognizer Cognitive Service swagger file public

### PR DESCRIPTION
**Pull request by olduroja**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    where: $
+    transform: >-
+      return $.
+        replace( /"context"/g, "\"context\"\n\"crypto/tls\"" ).
+        replace( /autorest\.NewClientWithUserAgent\(UserAgent\(\)\)/g, "autorest.NewClientWithOptions(autorest.ClientOptions{UserAgent: UserAgent(), Renegotiation: tls.RenegotiateFreelyAsClient})" )
+```
+
+```yaml $(go multi-api)
+batch:
+  - tag: release_1_0
```
Comment (jhendrixMSFT) [inline specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md]: The value `release_1_0` doesn't match the actual tag `package-1.0`, can you please fix?  I believe other config files have the same issue.

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]

File: specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+These settings apply only when `--go` is specified on the command line.
+
+```yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: inkrecognizer
+  clear-output-folder: true
+
+  directive:
+  # Replace the default client constructor with the one that enables TLS renegotiation.
```
Comment (jhendrixMSFT) [inline specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md]: Also can you please confirm that your service requires this functionality and it's not just a copy/paste artifact?

**Comment by sarangan12** [comment]
@jhendrixMSFT  Since I looked into the private repo PR, I am taking over this PR. But, I would require your feedback on the Go specific comment that you provided

**Review by sarangan12 — APPROVED** [review]

File: specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md (hunk 1 above)
Comment (jhendrixMSFT) [reply specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md]: @sarangan12 FYI this same issue is the cause of failure for the Ruby SDK build

Comment (olduroja) [reply specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md]: @jhendrixMSFT  I've fixed this in readme and readme.go.md. I don't see it in the ruby file though. Am I missing something?

File: specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md (hunk 2 above)
Comment (olduroja) [reply specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md]: Removing. Confirmed that it's not needed.

File: specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md (hunk 1 above)
Comment (jhendrixMSFT) [reply specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md]: Ruby is also `release_1_0` but now that you updated the README they all match.

File: specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+```yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: inkrecognizer
+  clear-output-folder: true
+  
+```
+
+```yaml $(go multi-api)
```
Comment (jhendrixMSFT) [inline specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md]: This should be ```yaml $(go) && $(multiapi)

**Comment by yangyuan** [comment]
Hi @olduroja,
Have you gotten the sign-off from API review board?

File: specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md (hunk 3 above)
Comment (olduroja) [reply specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md]: Fixed. Thanks

**Comment by olduroja** [comment]
@yangyuan  Yes, we have sign-off from the API review board.

File: specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  - tag: release_1_0
+
+```
+### Tag: release_1_0 and go
+
+These settings apply only when `--tag=release_1_0 --go` is specified on the command line.
+Please also specify `--go-sdks-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+```yaml $(tag) == 'release_1_0' && $(go)
+output-folder: $(go-sdks-folder)/services/preview/cognitiveservices/v1.0/$(namespace)
```
Comment (jhendrixMSFT) [inline specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md]: Extra 's', should be `$(go-sdk-folder)`

Comment (olduroja) [reply specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md]: fixed.

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Review by jhendrixMSFT — APPROVED** [review]

**Review by sarangan12 — APPROVED** [review]

**Pull request merged**